src/BloomExe/Book/Book.cs

```diff
@@ -581,10 +581,16 @@ private HtmlDom GetErrorDom()
 			return new HtmlDom(builder.ToString());
 		}
 
+		private bool IsDownloaded
+		{
+			get { return FolderPath.Contains(BookCollection.DownloadedBooksCollectionNameInEnglish); }
```

Comment:
Why not just use BookTransfer.DownloadFolder and test whether it is for sure the folder we use for Downloads? This COULD fail if the user made one with that name..